platform/net/zephyr/src/mender-http.c

```diff
@@ -58,6 +60,14 @@ static mender_http_config_t mender_http_config;
  */
 static void mender_http_response_cb(struct http_response *response, enum http_final_call final_call, void *user_data);
 
+/**
+ * @brief HTTP artifact response callback, invoked to handle data received
+ * @param response HTTP response structure
+ * @param final_call Indicate final call
+ * @param user_data User data, used to retrieve request context data
+ */
+static void artifact_response_cb(struct http_response *response, enum http_final_call final_call, void *user_data);
```

Comment:
For consistency with the above cb: `mender_artifact_response_cb`

Reply:
Actually, I think it should be the other way around -- no `mender_` prefix for `static` functions.